docs/getting-started/fundamentals-block-development/the-block-wrapper.md

```diff
@@ -0,0 +1,113 @@
+# The block wrapper
+
+Each block's markup is wrapped by a container HTML tag that needs to have the proper attributes to fully work in the Block Editor and to reflect the proper block's style settings when rendered in the Block Editor and the front end. As developers, when creating a custom block, we need to manually add these attributes to the markup using some of the tools provided by WordPress.
+
+Ensuring proper attributes to the block wrapper is especially important when using custom styling or features like `supports`. 
+
+<div class="callout callout-info">
+The use of <code>supports</code> generates a set of properties that need to be manually added to the wrapping element of the block so they're properly stored as part of the block data
+</div>
+
+A block can have three markups defined, each one of them with a specific target and purpose:
+
+- The one for the **Block Editor**, defined through a `edit` React component passed to `registerBlockType` when registering the block in the client. 
+- The one used to **save the block in the DB**, defined through a `save` React component passed to `registerBlockType` when registering the block in the client. 
```

Comment:
I've mentioned this in other review as well but `save` is not technically a React component but rather a function that return JSX. React components are [always uppercase](https://react.dev/learn/your-first-component#step-2-define-the-function).

Reply:
@ryanwelcher I have updated the texts to refer to `save` as a function rather than as a React component